Server/seeds/characters_1.js

```diff
@@ -0,0 +1,159 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('characters').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('characters').insert([
+        {character_name: 'Answer', game_id: 1,
+        summary: 'Adjutant to self-proclaimed President Chipp Zanuff. He is blessed with a preternatural memory, and can remember any facts after hearing them once. Some refer to him as a "Human Database".',
+         defense_modifier: 1.03, guts: 0, weight: '(100) Medium', stun_resistance: 55,
```

Comment:
### How are the base stats going to look unformatted? 
Is it going to be pulled in as a string -> `<p>{summary+stats}</p>`? Could get messy.
Maybe reconsider making another column for this data, even if you just stick it in a `<p>` tag later. Copy pasta!